mudirac/mudirac.xml

```diff
@@ -185,9 +205,24 @@ spec_expdec: $spec_expdec
             <option value="SPHERE"> finite size, uniformly charged spherical nucleus </option>
             <option value="FERMI2"> Fermi 2-term charge distribution </option>
         </param>
-        <param name="uehling_correction" type="boolean" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not" optional="true"/>
+        <conditional name="uehling_correction">
+            <param name="uehling_correction" type="select" display="radio" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not">
```

Comment:
```suggestion
            <param name="uehling_correction" type="select" display="radio" label="Uehling correction" help="whether to turn on the Uehling correction or not">
```

I _think_ value is redundant here, as `selected="true"` on the option below is what actually sets the default: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-inputs-param-option ?